docs/source/quickstart.rst

```diff
@@ -61,17 +61,6 @@ as those are the subset of Python language features CMake actually supports.
 Installing
```

Comment:
Is there a way to avoid duplicating these install instructions, *i.e.*, can we put them in a single place and link to them?

Reply:
A link is possible, but it would require splitting the instructions to a separate `installation.rst` file and then linking to it, as `include` directives are not allowed in Github's rendering engine. This means the instructions are not "front-and-center" on the README but a simple click would redirect users to them.

Reply:
I think a line like `[How to install](docs/source/installation.rst)` (or whatever the path is) on the main README is fine.